extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -251,6 +256,18 @@ require([
 				handleClick(event, category);
 			}
 		});
+
+		// Track clicks in the To-do List module
+		$('.community-page-todo-list-module-edit').on('mousedown touchstart', 'a', function (event) {
+			handleClick(event, 'community-page-todo-list-module');
+		});
+
+		$('.community-page-todo-list-module-content').on('mousedown touchstart', 'a', function (event) {
+			track({
+				category: 'community-page-todo-list-module',
+				label: event.delegateTarget.getAttribute('data-tracking'),
```

Comment:
Ah, I see, since that's the case, I probably wouldn't bother with using the HTML data attribute for the label, as you're not going to use more than one, right?